src/index.html

```diff
@@ -11,8 +11,59 @@
       rel="stylesheet"
       href="./styles/index.scss"
```

Comment:
The `href` attribute in the `link` tag should point to a CSS file, not a SCSS file. The browser cannot interpret SCSS. You need to compile your SCSS to CSS.